ui-v2/src/components/ui/icons/constants.ts

```diff
@@ -0,0 +1,51 @@
+import {
+	AlignVerticalJustifyStart,
+	BanIcon,
+	CheckIcon,
+	ChevronDownIcon,
+	ChevronLeft,
+	ChevronRight,
+	ChevronsLeft,
+	ChevronsRight,
+	CircleArrowOutUpRight,
+	ClockIcon,
+	ExternalLinkIcon,
+	Loader2,
+	MoreHorizontal,
+	MoreVerticalIcon,
+	PanelLeft,
+	PauseIcon,
+	PlayIcon,
+	PlusIcon,
+	SearchIcon,
+	ServerCrashIcon,
+	Variable,
+	X,
+} from "lucide-react";
+
+export const ICONS = {
+	AlignVerticalJustifyStart,
+	BanIcon,
+	CheckIcon,
+	ChevronDownIcon,
+	ChevronLeft,
+	ChevronRight,
+	ChevronsLeft,
+	ChevronsRight,
+	CircleArrowOutUpRight,
+	ClockIcon,
+	ExternalLinkIcon,
+	Loader2,
+	MoreHorizontal,
+	MoreVerticalIcon,
+	PanelLeft,
+	PauseIcon,
+	PlayIcon,
+	PlusIcon,
+	SearchIcon,
+	ServerCrashIcon,
+	Variable,
+	X,
+} as const;
+
+export type IconId = keyof typeof ICONS;
```

Comment:
It does, but they don't recommend using it because it drastically increased bundle size: https://lucide.dev/guide/packages/lucide-react#one-generic-icon-component